.azure-pipelines/official-images/official-build-barefoot.yml

```diff
@@ -0,0 +1,24 @@
+# Starter pipeline
```

Comment:
You can use the same yaml file for all build definitions, different definition uses different parameters.

Reply:
Maybe I can drop another PR for compare. 
This kind of change will remove template parameters limit, is hard to set default values, and is like classic Jenkins job. parameters are in Web config, not in code. I prefer not use the same yaml file.